src/.editorconfig

```diff
@@ -27,10 +27,9 @@ insert_final_newline = false
 dotnet_diagnostic.CA1068.severity = error
 dotnet_diagnostic.CA1710.severity = error
 dotnet_diagnostic.CA1711.severity = error
-dotnet_diagnostic.CA2007.severity = error
+dotnet_diagnostic.CA2007.severity = none
```

Comment:
Won't this be reset by repo sync?

Reply:
ugh, didn't think about that, is there a workaround for that?

Reply:
No there isn't. If want to make this change, you need to add an .editorconfig file to every project to disable it.

Reply:
Got it so the only (future) option would be to have different editor configs for "library" repos vs "non-library" repos where that setting could be different?

Reply:
There's no way to have that sort of variation given the way RepoSync is implemented.